mvnw.cmd

```diff
@@ -37,7 +37,7 @@
 @echo off
 @REM set title of command window
 title %0
-@REM enable echoing my setting MAVEN_BATCH_ECHO to 'on'
+@REM enable echoing by setting MAVEN_BATCH_ECHO to 'on'
 @if "%MAVEN_BATCH_ECHO%" == "on"  echo %MAVEN_BATCH_ECHO%
```

Comment:
Is there any real semantic difference between `MAVEN_BATCH_ECHO` and `MVNW_VERBOSE`? I'm not sure, if `MAVEN_BATCH_ECHO` should be used within this cmd file. Otherwise, what's the purpose of `MAVEN_BATCH_ECHO`?

Reply:
`MAVEN_BATCH_ECHO=on` is similar to running with `set -x` in Unix. It's also used by `mvn.cmd`.